vendor/bin/cexec.cmd

```diff
@@ -19,8 +19,8 @@ set "currenArgu=%~1"
 if /i "%currenArgu%" equ "/setPath" (
   :: set %flag_exists% shortcut
   endlocal
-  set "ccall=call ^"%~dp0cexec.cmd^""
-  set "cexec=^"%~dp0cexec.cmd^""
+  set "ccall=call %~dp0cexec.cmd"
```

Comment:
`^"call %~dp0cexec.cmd^"` is required in case a user puts cmder in a path with spaces in it and wants to us `cexec` or `ccall` functionality.

Reply:
![fresh-install](https://user-images.githubusercontent.com/4619155/90200529-7bc1c080-de1b-11ea-860c-47ff71b15581.png)

This change fixes the error in the screenshot.

Reply:
And this happens when Cmder starts? I have never seen this error or antly report of this error.

Reply:
Yup - on launch.